curriculum/challenges/english/24-b1-english-for-developers/learn-common-phrasal-verbs-and-idioms/685a94d8bf306e28027ca804.md

```diff
@@ -0,0 +1,80 @@
+---
+id: 685a94d8bf306e28027ca804
+title: Task 125
+challengeType: 22
+dashedName: task-125
+---
+
+<!-- (Audio) Bob: How about we start sending out the invites early next week? -->
+
+# --instructions--
+
+Listen to the audio and complete the sentence below.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`How about we start sending out the BLANK early next week?`
+
+## --blanks--
+
+`invites`
+
+### --feedback--
+
+This is a request to attend an event. Use the plural form.
+
+# --explanation--
+
+`Invite` is a short form for `invitation`, which is a request or offer for someone to attend an event or take part in an activity. It can be formal or informal and can be sent by letter, email, text, or verbally. `Invite` is often used informally to refer to a request for someone to attend an event. For example:
```

Comment:
```suggestion
`Invite` is a short form for `invitation`, which is a request or offer for someone to attend an event or take part in an activity. It can be formal or informal and can be sent by letter, email, text, or verbally. For example:
```

Reply:
I remove a repeated sentence.